server/src/completionUtil.ts

```diff
@@ -63,119 +67,219 @@ export const blockAttributes: CompletionItem[] = [
     insertTextFormat: 2,
     insertText: '@@unique([$0])',
-    documentation: 'Defines a composite unique constraint across fields.',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@@unique(_ fields: FieldReference[])',
+        '```',
+        '___',
+        'Defines a compound unique constraint for the specified fields.',
+        '',
+        '_@param_ `fields` A list of references',
+      ].join('\n'),
+    },
   },
   {
     label: '@@index([])',
     kind: CompletionItemKind.Property,
     insertTextFormat: 2,
     insertText: '@@index([$0])',
-    detail: '@@index(_ fields: Identifier[], name: String?)',
-    documentation: 'Defines an index for multiple fields.',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@@index(_ fields: FieldReference[])',
+        '```',
+        '___',
+        'Defines an index.',
+        '',
+        '_@param_ `fields` A list of references',
+      ].join('\n'),
+    },
   },
 ]
 
 export const fieldAttributes: CompletionItem[] = [
   {
     label: '@id',
     kind: CompletionItemKind.Property,
-    detail: '@id',
-    documentation:
-      'Defines the primary key. There must be exactly one field @id or block @id',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@id',
+        '```',
+        '___',
+        'Defines a single-field ID on the model.',
+      ].join('\n'),
+    },
   },
   {
     label: '@unique',
     kind: CompletionItemKind.Property,
-    detail: '@unique',
-    documentation: 'Defines the unique constraint.',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@unique',
+        '```',
+        '___',
+        'Defines a unique constraint for this field.',
+      ].join('\n'),
+    },
   },
   {
     label: '@map()',
     kind: CompletionItemKind.Property,
-    detail: '@map(_ name: String)',
     insertTextFormat: 2,
     insertText: '@map($0)',
-    documentation: 'Defines the raw column name the field is mapped to.',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@map(_ name: String)',
+        '```',
+        '___',
+        'Maps a field name from the Prisma schema to a different column name.',
+        '',
+        '_@param_ `name` The name of the target database column',
+      ].join('\n'),
+    },
   },
   {
     label: '@default()',
     kind: CompletionItemKind.Property,
-    detail: '@default(_ expr: Expr)',
     insertTextFormat: 2,
     insertText: '@default(0)',
-    documentation: 'Specifies a default value if null is provided.',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@default(_ expression: Expression)',
+        '```',
+        '___',
+        'Defines a default value for this field. `@default` takes an expression as an argument.',
```

Comment:
Does it not also take a string?

Reply:
According to the [Prisma Doc](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/data-model#overview-1), an expression is e.g. `5`, `true` or `now()`. This is also described in the param comment.